[2022-02-12 12:44:18,736] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: fhv_data_2_gcs_dag.bq_ext_2_part_task manual__2022-02-12T12:43:03.312404+00:00 [queued]>
[2022-02-12 12:44:18,755] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: fhv_data_2_gcs_dag.bq_ext_2_part_task manual__2022-02-12T12:43:03.312404+00:00 [queued]>
[2022-02-12 12:44:18,756] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-12 12:44:18,757] {taskinstance.py:1239} INFO - Starting attempt 2 of 3
[2022-02-12 12:44:18,758] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-12 12:44:18,769] {taskinstance.py:1259} INFO - Executing <Task(BigQueryInsertJobOperator): bq_ext_2_part_task> on 2022-02-12 12:43:03.312404+00:00
[2022-02-12 12:44:18,775] {standard_task_runner.py:52} INFO - Started process 612 to run task
[2022-02-12 12:44:18,779] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'fhv_data_2_gcs_dag', 'bq_ext_2_part_task', 'manual__2022-02-12T12:43:03.312404+00:00', '--job-id', '1060', '--raw', '--subdir', 'DAGS_FOLDER/fhv_data_2_gcs_dag.py', '--cfg-path', '/tmp/tmpctu31dac', '--error-file', '/tmp/tmp4y0vqhlj']
[2022-02-12 12:44:18,781] {standard_task_runner.py:77} INFO - Job 1060: Subtask bq_ext_2_part_task
[2022-02-12 12:44:18,853] {logging_mixin.py:109} INFO - Running <TaskInstance: fhv_data_2_gcs_dag.bq_ext_2_part_task manual__2022-02-12T12:43:03.312404+00:00 [running]> on host 853fb9201673
[2022-02-12 12:44:18,915] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fhv_data_2_gcs_dag
AIRFLOW_CTX_TASK_ID=bq_ext_2_part_task
AIRFLOW_CTX_EXECUTION_DATE=2022-02-12T12:43:03.312404+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-12T12:43:03.312404+00:00
[2022-02-12 12:44:18,918] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-02-12 12:44:18,919] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-12 12:44:18,929] {bigquery.py:1637} INFO - Inserting job ***_fhv_data_2_gcs_dag_bq_ext_2_part_task_2022_02_12T12_43_03_312404_00_00_7d3915d37c45b0ad3fce8de923204a10
[2022-02-12 12:44:47,456] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=fhv_data_2_gcs_dag, task_id=bq_ext_2_part_task, execution_date=20220212T124303, start_date=20220212T124418, end_date=20220212T124447
[2022-02-12 12:44:47,501] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-12 12:44:47,541] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
